### PR TITLE
Mock data refactor (KHO-221)

### PR DESCRIPTION
**Pull request by alexangione419**
All "mock" data uses real data. 
There are two classrooms, Spring and Fall 2025. 
We are all in both classrooms, there exist student teams on GH that we are also in
Both classrooms have an assignment that tie to real repos
There's a generic rubric in there too

# Checklist

- [ ] I have performed a self-review of my code
- [ ] I have reached out to another developer to review my code
- [ ] I acknowledge my responsibility to reinitialize the deployed DB if there are schema changes.
- [ ] New and existing unit tests pass locally with my changes
- [ ] All Lint and CI checks are passing

### CODE REVIEW

File: frontend/src/pages/Organizations/Select/index.tsx [hunk 1]
```diff
@@ -21,9 +21,11 @@ const OrganizationSelection: React.FC = () => {
         const data: IOrganizationsResponse = await getAppInstallations();
         //checking if data exists before populating and setting lists
         if (data.orgs_with_app) {
+          console.log(data.orgs_with_app)
           setOrgsWithApp(data.orgs_with_app);
         }
         if (data.orgs_without_app) {
+          console.log(data.orgs_without_app)
```
Comment (sebytremblay) [inline frontend/src/pages/Organizations/Select/index.tsx:28]: delete

File: backend/internal/handlers/organizations/organization.go [hunk 2]
```diff
@@ -193,6 +194,7 @@ func (service *OrganizationService) GetOrgTemplateRepos() fiber.Handler {
 				}
 				templateRepos = append(templateRepos, templateRepo)
 
+                fmt.Println(templateRepo)
```
Comment (sebytremblay) [inline backend/internal/handlers/organizations/organization.go:197]: delete

File: backend/internal/handlers/classrooms/assignments/assignments.go [hunk 3]
```diff
@@ -98,6 +99,7 @@ func (s *AssignmentService) createAssignment() fiber.Handler {
 		if err != nil {
 			return err
 		}
+        fmt.Println(assignmentData)
```
Comment (sebytremblay) [inline backend/internal/handlers/classrooms/assignments/assignments.go:102]: delete

File: backend/database/migrations/002_mock_data.sql [hunk 4]
```diff
@@ -1,212 +1,71 @@
...
-(4, 4, 'adalovelace/encryption-algorithms', '2023-05-01 23:59:59', 45, 40, '2023-05-05 16:00:00', 'GRADE_PUBLISHED', NOW()),
-(5, 5, 'mhamilton/mobile-app', '2023-06-01 23:59:59', 18, 15, '2023-06-05 18:00:00', 'GRADE_PUBLISHED', NOW()),
-(6, 6, 'kennysmith/ai-chatbot', '2023-07-01 23:59:59', 50, 45, '2023-07-05 20:00:00', 'GRADE_PUBLISHED', NOW()),
-(7, 7, 'bliskov/website-dev', '2023-08-01 23:59:59', 28, 25, '2023-08-05 22:00:00', 'GRADE_PUBLISHED', NOW()),
-(8, 8, 'linustorvalds/cloud-deployment', '2023-09-01 23:59:59', 40, 35, '2023-09-05 12:00:00', 'GRADE_PUBLISHED', NOW()),
-(9, 9, 'mhamilton/cybersecurity-analysis', '2023-10-01 23:59:59', 48, 45, '2023-10-05 14:00:00', 'GRADE_PUBLISHED', NOW()),
-(10, 10, 'kennysmith/software-design-patterns', '2023-11-01 23:59:59', 38, 35, '2023-11-05 16:00:00', 'GRADE_PUBLISHED', NOW()),
-(11, 1, 'kenny-assignment-josevaca1231', '2023-04-01 23:59:59', 40, 35, '2023-04-05 14:00:00', 'GRADE_PUBLISHED', NOW()),
-(12, 1, 'kenny-assignment-josevaca1231', '2023-04-01 23:59:59', 40, 35, '2023-04-05 14:00:00', 'GRADE_PUBLISHED', NOW());
-SELECT setval('student_works_id_seq', (SELECT MAX(id) FROM student_works));
```
Comment (sebytremblay) [inline backend/database/migrations/002_mock_data.sql:159]: Can you update the student works as well, please?

File: backend/database/migrations/002_mock_data.sql [hunk 5]
```diff
@@ -1,212 +1,71 @@
...
 
--- Insert into classroom_membership
+-- Classroom Membership Data
 INSERT INTO classroom_membership (user_id, classroom_id, classroom_role, created_at, status)
 VALUES
-(1, 1, 'PROFESSOR', NOW(), 'ACTIVE'),
+(1, 1, 'TA', NOW(), 'ACTIVE'),
 (2, 1, 'PROFESSOR', NOW(), 'ACTIVE'),
 (3, 1, 'STUDENT', NOW(), 'ACTIVE'),
+(4, 1, 'TA', NOW(), 'ACTIVE'),
```
Comment (nandini-ghosh) [inline backend/database/migrations/002_mock_data.sql:27]: Mix up student and TA and professor roles amongst the mock users ty

**Review by sebytremblay — APPROVED** [review]
lgtm

**Pull request merged**